packages/components/src/styles/copyright/copyright.stories.ts

```diff
@@ -0,0 +1,45 @@
+import '../../solid-components';
+
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+
+const { argTypes, parameters } = storybookDefaults('sd-copyright');
+const { overrideArgs } = storybookHelpers('sd-copyright');
+const { generateTemplate } = storybookTemplate('sd-copyright');
+
+/**
+ * Add a copyright at the bottom of a image.
+ */
+export default {
+  title: 'Styles/sd-copyright',
+  component: 'sd-copyright',
+  parameters: {
+    ...parameters
+  },
+  args: overrideArgs(
+    {
+      type: 'slot',
+      name: 'default',
+      value: `<img src="./placeholders/generic.jpg" alt="A generic placeholder jpg" />`
+    },
+    {
+      '--copyright': '© 2024 Solid Design System'
+    }
+  ),
+  argTypes
+};
+
+export const Default = {
+  parameters: {
+    controls: {
+      disable: true
+    }
+  },
+  render: (args: { [k: string]: any }) => {
+    return generateTemplate({
+      options: {
+        templateContent: `<div class="%CLASSES% max-w-xl" style="--copyright: '${args['--copyright']}';">%SLOT%</div>`
```

Comment:
Do I use the args the right way here, or how would you allow the --copyright property to be edited in storybook?

Reply:
If it works, then it should be fine!